Makefile

```diff
@@ -78,10 +78,13 @@ container-push-operator:
 container-push-registry:
 	docker push $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG)
 
+csv-generator: gen-operator-sdk
+	./build/make-csv-generator.sh
+
 gen-operator-sdk:
 	./hack/gen-operator-sdk.sh ${OPERATOR_SDK_VERSION}
 
-manifests: gen-operator-sdk
+manifests: csv-generator gen-operator-sdk
```

Comment:
gen-operator-sdk is already under cnv-generator. can be removed

Reply:
fixed